python/lsst/ts/hexrotcomm/base_csc.py

```diff
@@ -663,12 +648,18 @@ async def enable_controller(self):
                 await self.run_command(
                     code=self.CommandCode.SET_STATE, param1=command_param
                 )
+                # Waiting for the controller state is not necessary, but it
+                # makes sure that the CSC publishes all intermediate
+                # controller states in the controllerState event,
+                # making the data more predictable and easier to understand.
                 await self.wait_controller_state(resulting_state)
                 current_state = resulting_state
             except Exception as e:
-                raise salobj.ExpectedError(
+                errmsg = (
                     f"SET_STATE command with param1={command_param!r} failed: {e!r}"
-                ) from e
+                )
+                self.log.error(errmsg)
+                raise salobj.ExpectedError(errmsg) from e
```

Comment:
Please consider to add the doc string of `ExpectedError` in the `Raises` section of function description.